crates/harness/src/monolith.rs

```diff
@@ -391,7 +426,10 @@ impl MonolithBuilder {
     }
 
     pub async fn build(self, ctx: &TestRunner) -> Monolith {
-        let mut monolith = Monolith::new(ctx).await.unwrap();
+        let mut monolith =
+            Monolith::with_behavior(ctx, self.behavior.unwrap_or(Box::new(BehaviorManual)))
```

Comment:
Why do pointers need to be in a box?

Reply:
A `Box` is just a heap allocated pointer. You have to do this because `dyn Trait` is not sized, and the compiler has to know how big each type is in order to allocate memory for it.